### PR TITLE
[CortexXpanse] EXPANDR-9701: Layout Updates

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9701

## Description
Updating layout to be more valuable to vulnerability analysts.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@BigEasyJ / @capanw , please review when possible

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment_README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
 
-GridFieldSetup
+* Set
+* GridFieldSetup
 
 ### Commands
 
-* asm-list-external-service
-* asm-list-asset-internet-exposure
+* extractIndicators
```
Comment (BigEasyJ) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment_README.md:22]: NIT: Alphabetical order

File: Packs/CortexXpanse/Classifiers/classifier-Xpanse_-_Incoming_Mapper.json [hunk 2]
```diff
@@ -6,195 +6,198 @@
...
+                                    }
+                                },
+                                "operator": "SetIfEmpty"
+                            }
+                        ]
+                    }
+                },
                 "Xpanse Category": {
                     "simple": "asm_alert_categories"
+                },
```
Comment (BigEasyJ) [inline Packs/CortexXpanse/Classifiers/classifier-Xpanse_-_Incoming_Mapper.json:163]: NIT: This is the only new change in the file

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 3]
```diff
@@ -133,32 +128,31 @@ tasks:
...
+    taskid: 41e5b734-11c8-45d6-83cd-dd04e1f3980b
     type: regular
     task:
-      id: 5e18d026-6149-4f5c-858f-c7995a5b3550
+      id: 41e5b734-11c8-45d6-83cd-dd04e1f3980b
       version: -1
       name: Search external service information
-      description: Get a list of all your external services filtered by the remote IP of the alert.
-      script: '|||asm-list-external-service'
+      description: Get service details according to the service ID.
```
Comment (BigEasyJ) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:137]: Is `asm-get-external-service` going to hang, does it need a sleep or poll?

**Review by BigEasyJ — APPROVED** [review]

**Review by capanw — APPROVED** [review]

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml (hunk 3 above)
Comment (johnnywilkes) [reply Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:137]: Was hoping that https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8203 should make it so this isn't necessary.  We can add it later if needed

File: Packs/CortexXpanse/Classifiers/classifier-Xpanse_-_Incoming_Mapper.json (hunk 2 above)
Comment (johnnywilkes) [reply Packs/CortexXpanse/Classifiers/classifier-Xpanse_-_Incoming_Mapper.json:163]: "Xpanse Attack Surface Rule Name" is the only change, not sure the entire diff

File: Packs/CortexXpanse/pack_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Xpanse",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.0.26",
+    "currentVersion": "1.1.0",
```
Comment (BigEasyJ) [inline Packs/CortexXpanse/pack_metadata.json:5]: Love it!

**Comment by johnnywilkes** [comment]
@ssokolovich / @Shellyber , this has been approved by Xpanse.  Please merge

@ShirleyDenkberg , please review

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 5]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+                value:
+                  simple: n/a
+            operator: SetIfEmpty
     separatecontext: false
-    continueonerrortype: ""
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:344]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 6]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
-          accessor: protocol
+                  simple: n/a
+            operator: SetIfEmpty
     separatecontext: false
-    continueonerrortype: ""
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:396]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 7]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+                value:
+                  simple: n/a
+            operator: SetIfEmpty
     separatecontext: false
-    continueonerrortype: ""
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:448]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 8]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+                value:
+                  simple: ', '
+            operator: join
     separatecontext: false
-    continueonerrortype: ""
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:499]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 9]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+              defaultValue:
+                value:
+                  simple: n/a
+            operator: SetIfEmpty
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:772]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 10]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:973]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 11]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1025]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 12]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+          accessor: first_observed
+          root: ASM.ExternalService
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1073]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 13]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+          accessor: last_observed
+          root: ASM.ExternalService
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1121]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 14]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "108"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to service grid field.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1223]:
```suggestion
      description: Determines if there are time entries to add to the service grid field.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 15]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "120"
+      "yes":
+      - "103"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1372]:
```suggestion
      description: Determines if there are time entries to add to the asset grid field.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 16]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+              defaultValue:
+                value:
+                  simple: n/a
+            operator: SetIfEmpty
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1676]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 17]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1902]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 18]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2072]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 19]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "126"
+      "yes":
+      - "121"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2128]:
```suggestion
      description: Determines if there are time entries to add to the asset grid field.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 20]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+              defaultValue:
+                value:
+                  simple: n/a
+            operator: SetIfEmpty
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2339]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 21]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "89"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a asset id to enrich.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2386]:
```suggestion
      description: Determines if there is an asset ID to enrich.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 22]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "88"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a service id to enrich.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2431]:
```suggestion
      description: Determines if there is a service ID to enrich.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 23]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "131"
+      "yes":
+      - "110"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2492]:
```suggestion
      description: Determines if there are time entries to add to the asset grid field.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 24]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      val7:
+        simple: n/a
+      val8:
+        simple: n/a
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2610]:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Example of command:
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment_README.md [hunk 25]
```diff
@@ -1,31 +1,32 @@
...
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
 ### Integrations
 
-Cortex Xpanse
+* Cortex Xpanse
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment_README.md:13]:
```suggestion
Cortex Xpanse
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 26]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "114"
     note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have an asset of type certificate.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:547]:
```suggestion
      description: Determines if there is an asset of type certificate.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 27]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "130"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have an asset of type responsive IP.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:820]:
```suggestion
      description: Determines if there is an asset of type responsive IP.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 28]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to service grid field.
+      id: 064f771f-dff0-4705-8fca-91b07804d0ed
+      iscommand: false
+      name: Do we have time entries?
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1226]:
```suggestion
      name: Are there time entries?
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 29]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
+      id: e6a098c6-5cdc-4fdd-810d-b866d2d7da53
+      iscommand: false
+      name: Do we have time entries?
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1375]:
```suggestion
      name: Are there time entries?
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 30]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "124"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have an asset of type domain.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1418]:
```suggestion
      description: Determines if there is an asset of type domain.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 31]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+      - "38"
+      "yes":
+      - "122"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have an asset of type cloud.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:1464]:
```suggestion
      description: Determines if there is an asset of type cloud.
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 32]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
+      id: 4b745d05-e52e-4c0f-8d97-2fcc071f4dec
+      iscommand: false
+      name: Do we have time entries?
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2131]:
```suggestion
      name: Are there time entries?
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml [hunk 33]
```diff
@@ -299,240 +295,2362 @@ tasks:
...
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if we have a time entries to add to asset grid field.
+      id: f81c2323-d804-4a83-8972-b59dc47930ae
+      iscommand: false
+      name: Do we have ip_range entries?
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml:2495]:
```suggestion
      name: Are there ip_range entries?
```

**Comment by ShirleyDenkberg** [comment]
@BigEasyJ @capanw @ssokolovich @Shellyber Doc review completed.

**Comment by johnnywilkes** [comment]
@ssokolovich  @Shellyber  Are these validation errors [legitimate](https://app.circleci.com/pipelines/github/demisto/content/362989/workflows/5e88aed8-4b49-4b17-b7e9-33d3da10b124/jobs/748508)?

**Comment by Shellyber** [comment]
@johnnywilkes No. It seems that we have some problems in our build, we're working on fixing them

**Review by Shellyber — COMMENTED** [review]
Looking good for my part.
@ssokolovich Please approve as well

**Review by Shellyber — APPROVED** [review]

**Pull request merged**